src/js/pouch.js

```diff
@@ -0,0 +1,70 @@
+// Utility functions to add pouch to an existing express instance
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.pouch");
+
+var PouchDB        = require("pouchdb");
+var memdown        = require("memdown");
+var expressPouchdb = require("express-pouchdb");
+
+gpii.pouch.init = function(that) {
+    var MemPouchDB = PouchDB.defaults({db: memdown });
+
+    if (that.model.databases && Object.keys(that.model.databases).length > 0) {
+        Object.keys(that.model.databases).forEach(function(key){
+            var dbConfig = that.model.databases[key];
+            var db = new MemPouchDB(key);
+            if (dbConfig.data) {
+                var data = require(dbConfig.data);
+                db.bulkDocs(data);
+            }
+        });
+    }
+
+    that.expressPouchdb = expressPouchdb(MemPouchDB);
+
+    // Sadly, expressPouchdb does not provide a ready event or callback.  We have to manually enforce a waiting period by waiting to fire our "started" event.
+    setTimeout(function() { that.events.started.fire(); },500);
```

Comment:
It seems like the return value from express-pouchdb is an express() - https://github.com/pouchdb/express-pouchdb/blob/master/lib/index.js#L80 - so, in itself, it should be ready synchronously. The waiting we may require might be i) any server we attach it to - which we would wait for ourselves, or ii) the MemPouchDB instance - in either case we should avoid this hardwired timeout